language.md

```diff
@@ -411,9 +428,31 @@ mySqrt a =               # If you pass `a = -1`,
 
 ## Pattern Matching
 
-TODO: Write something
+Candy supports structural pattern matching using the match operator `%`.
+
+```candy
+match (foo 5) %
```

Comment:
This looks like `maybe` is also part of the necessary syntax. Maybe call a different function?